lib/mimemail.ex

```diff
@@ -70,13 +71,13 @@ defmodule MimeMail do
     case mail.headers[:'content-type'] do
       {"text/"<>_=type,params}-> 
         headers = Keyword.drop(mail.headers,[:'content-type',:'content-transfer-encoding']) ++[
-          'content-type': {type,Map.put(params,:charset,"utf-8")},
-          'content-transfer-encoding': "quoted-printable"
+          {:"content-type", {type,Map.put(params,:charset,"utf-8")}},
+          {:"content-transfer-encoding", "quoted-printable"}
```

Comment:
```suggestion
          "content-type": {type,Map.put(params,:charset,"utf-8")},
          "content-transfer-encoding": "quoted-printable"
```

this is accepted by the formatter